src/frontend/templates/pages/games_list.html

```diff
@@ -0,0 +1,152 @@
+{% extends 'base.html' %}
+
+{% block title %}
+Games for {{network.name}}
+{% endblock %}
+
+{% block content %}
+
+{% if kind == "training" %}
+<h1 class="title">Training Games for {{network.name}}</h1>
+{% else %}
+<h1 class="title">Rating Games for {{network.name}}</h1>
+{% endif %}
+
+{% if kind == "training" %}
+<div class="pagination">
+  <span class="step-links">
+    {% if page_obj.has_previous %}
+    <a href="{% url 'training_games_list' run.name network.name 1 %}">&laquo; First</a>
+    <a href="{% url 'training_games_list' run.name network.name page_obj.previous_page_number %}">Prev</a>
+    {% else %}
+    &laquo; First
+    Prev
+    {% endif %}
+
+    <span class="current">
+      Page {{ page_obj.number }} of {{ page_obj.paginator.num_pages }}.
+    </span>
+
+    {% if page_obj.has_next %}
+    <a href="{% url 'training_games_list' run.name network.name page_obj.next_page_number %}">Next</a>
+    <a href="{% url 'training_games_list' run.name network.name page_obj.paginator.num_pages %}">Last &raquo;</a>
+    {% else %}
+    Next
+    Last &raquo;
+    {% endif %}
+  </span>
+</div>
+{% else %}
+<div class="pagination">
+  <span class="step-links">
+    {% if page_obj.has_previous %}
+    <a href="{% url 'rating_games_list' run.name network.name 1 %}">&laquo; First</a>
+    <a href="{% url 'rating_games_list' run.name network.name page_obj.previous_page_number %}">Prev</a>
+    {% else %}
+    &laquo; First
+    Prev
+    {% endif %}
+
+    <span class="current">
+      Page {{ page_obj.number }} of {{ page_obj.paginator.num_pages }}.
+    </span>
+
+    {% if page_obj.has_next %}
+    <a href="{% url 'rating_games_list' run.name network.name page_obj.next_page_number %}">Next</a>
+    <a href="{% url 'rating_games_list' run.name network.name page_obj.paginator.num_pages %}">Last &raquo;</a>
+    {% else %}
+    Next
+    Last &raquo;
+    {% endif %}
+  </span>
+</div>
+{% endif %}
+
+<table>
```

Comment:
You can use the css for table

Reply:
https://bulma.io/documentation/elements/table/
![Uploading image.png…]()

Reply:
Done!